Project.toml

```diff
@@ -1,25 +1,25 @@
 name = "PlanetaryEphemeris"
 uuid = "d83715d0-7e5f-11e9-1a59-4137b20d8363"
 authors = ["Jorge A. Pérez Hernández", "Luis Benet"]
-version = "0.2.1"
+version = "0.3.0"
 
 [deps]
 AutoHashEquals = "15f4f7f2-30c1-5605-9d31-71845cf9641f"
```

Comment:
Should we add `ArgParse` as a dependency?

Reply:
I have not included `ArgParse` as a dependency since it is not used in the source code, only in the scripts.

Reply:
I think we have to include it somewhere, either here, or e.g. in `scripts/Project.toml`; not doing it will throw an error because `ArgParse` will not be recognized.

Reply:
Since the main use of `PlanetaryEphemeris.jl` is to run `scripts/integrate_ephemeris.jl` and it is a very lightweight package I have added `ArgParse` as a dependency.